lib/matplotlib/tests/test_axes.py

```diff
@@ -7902,6 +7902,89 @@ def test_ylabel_ha_with_position(ha):
     assert ax.yaxis.get_label().get_ha() == ha
 
 
+@image_comparison(baseline_images=['axes_labels_position'], remove_text=True,
```

Comment:
I don't think you want to remove the text from this test

Also why are you doing this w a bunch of subfigures?  Seems un necessary and it's usually better not to cross test features in this manner.

Reply:
Thanks @jklymak - should the plots then all be separate? Subfigures seemed the easiest way to get it all in one image for testing.

Reply:
Just use subplots 

I hadn't notice that you'd removed all ticks. In that case I guess you can remove all text too. Otoh it makes the tests harder to parse.